perun-core/src/main/java/cz/metacentrum/perun/core/bl/ResourcesManagerBl.java

```diff
@@ -219,7 +219,17 @@
 	boolean isUserAllowed(PerunSession sess, User user, Resource resource);
 
 	/**
-	 * Returns true if the group is assigned to the current resource, false otherwise.
+	 * Returns true if the group is assigned to the current resource with ACTIVE status, false otherwise.
+	 * @param sess
+	 * @param group
+	 * @param resource
+	 * @return true if the group is assigned to the current resource with active status.
+	 * @throws InternalErrorException
+	 */
+	boolean isGroupAssignedActive(PerunSession sess, Group group, Resource resource);
```

Comment:
I like the way you created this new method. But I would prefer `isGroupAssigned` to remain as it was. The reason is that we have many other methods like `isUserAssigned`, etc. where _assigned_ means that there is an ACTIVE assignment and it could become a little confusing if we changed it only here. Instead, a method checking if there is a group-resource assignment in any state could be named `hasGroupAssignment(Resource, Group)` or `groupResourceAssignmentExists(Group, Resource)`. What do you think @Kuliak @Johaney-s ?

Reply:
sure, I will change it, so it will not be confusing for anyone.